fix: tailwindcss & prettier conf (#636)
sveltejs/cli: .changeset/tangy-friends-camp.md

---
'sv': patch
---

> fix(add): when tailwindcss & prettier add `tailwindStylesheet` in `.prettierrc`

Reviewer: ```suggestion
fix(tailwindcss): ensure `tailwindStylesheet` is added to `.prettierrc`, when applicable
```
